WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -31,15 +31,191 @@
 from types import IntType, LongType, ListType
 import threading
 
+from sqlalchemy.sql.schema import ForeignKey, PrimaryKeyConstraint
+from sqlalchemy.sql.sqltypes import DateTime
+from sqlalchemy.orm import session, sessionmaker, scoped_session, mapper
+from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy import create_engine, Table, Column, MetaData, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+import datetime
+import time
+
 DEBUG = 1
 
+metadata = MetaData()
+Base = declarative_base()
+
 #############################################################################
 class PilotAgentsDB( DB ):
 
-  def __init__( self, maxQueueSize = 10 ):
+  def __init__( self ):
 
-    DB.__init__( self, 'PilotAgentsDB', 'WorkloadManagement/PilotAgentsDB', maxQueueSize )
+    DB.__init__( self, 'PilotAgentsDB', 'WorkloadManagement/PilotAgentsDB' )
     self.lock = threading.Lock()
+    
+    self.__initializeConnection('WorkloadManagement/PilotAgentsDB')
+    resp = self.__initializeDB()
+    if not resp['OK']:
+      raise Exception("Couldn't create tables: " + resp['Message'])
+  
+##########################################################################################
+  def __initializeConnection(self, dbPath):
+      
+      self.engine = create_engine( 'mysql://%s:%s@%s:%s/%s' 
+                                   %( self.dbUser, self.dbPass, self.dbHost, self.dbPort, self.dbName ),
+                                   pool_recycle = 3600, echo_pool = True )
+      self.sqlalchemySession = scoped_session( sessionmaker( bind = self.engine ) )
+      self.inspector = Inspector.from_engine( self.engine )
+
+##########################################################################################
+  def __initializeDB( self ):
+    
+    tablesInDB = self.inspector.get_table_names()
+    
+    if not 'PilotsUUIDtoID' in tablesInDB:
+      try:
+        PilotsUUIDtoID.__table__.create( self.engine )
+      except Exception, e:
+        return S_ERROR(e)
+    else:
+      gLogger.debug("Table PilotsUUIDtoID exists")
+      return S_OK()
+    
+    if not 'PilotsLogging' in tablesInDB:
+      try:
+        PilotsLogging.__table__.create( self.engine )
+      except Exception, e:
+        return S_ERROR(e)
+    else:
+      gLogger.debug("Table PilotsLogging exists")
+
+##########################################################################################
+  def addPilotsLogging(self, pilotUUID, status, minorStatus, timeStamp, source):
+    "Add new pilot logging entry"
```

Comment:
Minor comment: for docstrings always use triple quoters https://www.python.org/dev/peps/pep-0257/ (with """ ... """)